docs/10_firebase_sdk_configuration.md

```diff
@@ -0,0 +1,121 @@
+# Firebase SDK configuration
+
+> Summary of the proposed change
+
+Describes how to add a manual configuration of the Firebase SDK to the Flutter Web application.
+
+# References
+
+> Link to supporting documentation, GitHub tickets, etc.
+
+- [Firebase console](https://console.firebase.google.com/)
+
+# Motivation
+
+> What problem is this project solving?
+
+Explains how to add manual configuration of the Firebase SDK to the Flutter Web application.
```

Comment:
maybe the information in `docs/10_firebase_sdk_configuration.md` can be consolidated into `docs/09_firebase_deployment.md`?